docs/checklist_templates/eip_testing_checklist_template.md

```diff
@@ -0,0 +1,286 @@
+# EIP Execution Layer Testing Checklist Template
+
+Depending on the changes introduced by an EIP, the following template is the minimum baseline to guarantee test coverage of the Execution Layer features.
+
+## General
+
+- [ ] Code coverage
+    - [ ] Run produced tests against [EELS](https://github.com/ethereum/execution-specs) and verify that line code coverage of new added lines for the EIP is 100%, with only exeptions being unreachable code lines.
+    - [ ] Optional - Run against a second client and verify sufficient code coverage over new code added for the EIP.
+- [ ] Fuzzing
+    - [ ] TBD
+
+## New Opcode
+
+The EIP introduces one or more new opcodes to the EVM.
+
+### Test Vectors
+
+- [ ] Memory expansion
+    - [ ] Verify that the opcode execution results in the correct memory expansion, being by offset or size or interaction of both parameters (Size of zero should never result in memory expansion, regardless of offset value). Test at least the following memory expansion sizes
+        - [ ] Zero bytes expansion
+        - [ ] Single byte expansion
+        - [ ] 31 bytes expansion
+        - [ ] 32 bytes expansion
+        - [ ] 33 bytes expansion
+        - [ ] 64 or more bytes expansion
+- [ ] Stack over/underflows
+    - [ ] If the opcode pushes one or more items to the stack, and the opcode pushes more elements than it pops, verify that the opcode execution results in exeptional abort when pushing elements to the stack would result in the stack having more than 1024 elements.
+    - [ ] If the opcode pops one or more items to the stack, or it has a minimum stack height of one or more, verify that the opcode execution results in exeptional abort then stack has 1 less item than the minimum stack height expected.
+- [ ] Execution context
+    - [ ] Normal call
```

Comment:
- in create/create2  constructor.
- in transaction constructor.

Reply:
Thanks! This should be covered by lines 39-42.